data/json/items/magazine/weldgas.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "tinyweldtank",
+    "type": "MAGAZINE",
+    "name": "small welding tank",
+    "description": "A compact 5-litre steel cylinder designed for holding pressurised welding gases.  It is marked with several faded and now unreadable symbols.",
```

Comment:
It has 1 liter of volume itself. Also spelling - rest of the game spells it "liters" not "litres".